src/pages/Details.js

```diff
@@ -0,0 +1,26 @@
+import useLaunchList from '../hooks/useLaunchList';
+import { Card, Error, Loader, RocketDetails, RocketName } from './styledComponents';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
import { Card, Error, Loader, RocketDetails, RocketName } from "./styledComponents";
```